licenses/cla-individual.md

```diff
@@ -222,3 +222,5 @@ Andreas Abeck, @andreasabeck 2015/03/18
 Hong-Yi Dai, @reflectionalist 2015/04/04
 
 heroboy, @heroboy 2015/04/19
+
+vouko, @vouko 2015/05/02
```

Comment:
Hi @vouko thanks for your patience during this discussion.

@pmario @felixhayashi one response is to question whether we really need to have a CLA. At the time I started TW5 they seemed to be industry best practice, but now I'm seeing more voices arguing against them.

http://ebb.org/bkuhn/blog/2014/06/09/do-not-need-cla.html

> Ultimately, CLAs are much ado about nothing. Lawyers are trained to zealously represent their clients, and as such they often seek to an outcome that maximizes leverage of clients' legal rights, but they typically ignore the other important benefits that are outside of their profession. The most ardent supporters of CLAs have yet to experience first-hand the arduous daily work required to manage a queue of incoming FLOSS contributions. Those of us who have done the latter easily see that avoiding additional barriers to entry is paramount. While a beautifully crafted CLA — jam-packed with legalese that artfully shifts all the blame off to the contributors — may make some corporate attorneys smile, but I've never seen such bring anything but a frown and a sigh from FLOSS developers.